babel_godot.py

```diff
@@ -4,7 +4,7 @@
 __version__ = '0.2'
 
 
-_godot_node = re.compile(r'^\[node name="([^"]+)" (:?type="([^"]+)")?')
+_godot_node = re.compile(r'^\[(sub_)?resource|^\[node name="([^"]+)" (:?type="([^"]+)")?')
```

Comment:
I think it should be `(?:sub)` no? I guess you used your code only on resource files and not on scenes?

Reply:
I tried with both resources and scenes and it worked. I'm not very familiar with regex extensions, though. So, I've added what you asked.